### ISSUE TITLE
[BUG] H026 false positive for data-field="id"

### ISSUE THREAD

**Issue by jayvdb** [djlint/djLint#98]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3.7
 - djlint version: 0.5.6

## Issue

The fix for https://github.com/Riverside-Healthcare/djLint/issues/89 was very effective, however I have one last set of H026 false positives, where there is an attribute `data-field="id" . e.g.

```html
                <th data-field="id" style="display: none">ID</th>
```
